foundation_cms/templates/patterns/blocks/themes/default/text_image_block__narrow.html

```diff
@@ -1,3 +1,34 @@
-{% extends "./text_image_block.html" %}
```

Comment:
Similar concern --

Quite a few changes were made to this file. If they're intentional for this PR, could you please set up a review app page (outside of NP) so we can confirm there are no regressions?